tools/accuracy_checker/accuracy_checker/adapters/retinaface.py

```diff
@@ -62,6 +70,11 @@ def process(self, raw, identifiers=None, frame_meta=None):
                 keep = NMS.nms(x_mins, y_mins, x_maxs, y_maxs, scores, 0.5, False)
                 proposals_list.extend(proposals[keep])
                 scores_list.extend(scores[keep])
+                if self.type_scores_output:
+                    type_scores = raw_predictions[self.type_scores_output[_idx]][batch_id]
```

Comment:
it can be done in separated method, and lanmarks getting too

Reply:
Done